compiler/rustc_target/src/spec/linux_kernel_base.rs

```diff
@@ -9,6 +9,7 @@ pub fn opts() -> TargetOptions {
         // don't use probe-stack=inline-asm until rust#83139 and rust#84667 are resolved
         stack_probes: StackProbeType::Call,
         frame_pointer: FramePointer::Always,
+        executables: false,
         position_independent_executables: true,
```

Comment:
@joshtriplett 
Is this intentional?
The target doesn't support executables in general, but somehow supports PIC executables?

Reply:
Cc @ojeda

Reply:
Thanks for the ping Josh. We are not using the builtin target at the moment, so it should be fine.

As for the custom ones we generate, we don't set `executables` because the default was `false`, so switching it to `true` should not break anything (and we can always set it back to `false` on our side).

Reply:
As for `position_independent_executables`, we also don't set it (so it is `false` for us in the ones we generate). I guess it could be changed to `false` here (but we don't use the builtin target in any case).